01_Exercises/main.py

```diff
@@ -0,0 +1,74 @@
+# LAB 1
+# Calculati suma a N numere naturale.
+def sum_numbers(*numbers):
+    sum = 0
+    find = 0
+
+    for check in numbers:
+        if check < 0:
+            find = -1
+
+    for num in numbers:
+        sum = sum + num
+
+    if find < 0:
+        print("Choose only natural numbers")
+    else:
+        return sum
+
+
+print(sum_numbers(-2, 3))
+
+
+# Verificați daca un număr citit de la tastatura este prim.
+def only_prime_numbers(number):
+    divisors = []
+
+    for i in range(1, number + 1):
+        if number % i == 0:
+            divisors.append(i)
+
+    if number == 0 or number == 1:
+        print("This is NOT a prime number")
+    elif len(divisors) > 2:
+        print("This is NOT a prime number")
+    else:
+        print("This is a prime number")
+
+
+only_prime_numbers(13)
+
+
+# Calculați cel mai mare divizor comun a doua numere.
+def highest_divisors(a, b):
+    divisors_a = []
+    divisors_b = []
+    highest_divisor = []
+
+    for i in range(1, a):
```

Comment:
Trebuie pus `for i in range(1, a + 1):` la fel si la urmatorul for pentru ca divizorul unui numar este si el insusi. `highest_divisors(50, 100) ` o sa crape pentru ca cel mai mare divizor comun e 50

Reply:
Got it.
It does make sense.